amd64/Dockerfile

```diff
@@ -5,6 +5,10 @@ COPY nats-streaming-server /nats-streaming-server
 # Expose client and management ports
 EXPOSE 4222 8222
 
+# Run as a numbered non-root user for secure environments
+RUN addgroup -S 53000 && adduser -D -S -G 53000 53000
```

Comment:
Not sure there would be an `addgroup` command inside of a scratch container, does this work? I think you would get a no such file error and image fail to build

Reply:
you might be right actuallly. I could go back to what I had before without adding the group 🤔

Reply:
Confirmed and went back to the original stuff.